doc/reference/configuration/configuration_reference.rst

```diff
@@ -730,8 +730,46 @@ This section describes options related to providing connection settings to a :re
     | Default: 3
     | Environment variable: TT_CONFIG_STORAGE_TIMEOUT
 
+..  _configuration_reference_console:
 
+console
+-------
+
+Tarantool output can be written to a file or Unix socket.
```

Comment:
These two options (`console.{enabled,socket}`) are in fact a wrapper[^1] around the [console.listen](https://www.tarantool.io/en/doc/latest/reference/reference_lua/console/#console-listen) call with the following nuances:

* Only a Unix domain socket is allowed. A TCP socket can't be configured this way[^2].
* `console.socket` is a file path, without any `unix:` or `unix/:` prefixes[^3].
* If the file path is a relative path, it is interpreted relative to `process.work_dir` -- consistently with all the file paths in the declarative configuration.
* Enabled by default.

I don't like 'Tarantool output' in the current wording, because it is unclear what it is (logs, stored data, network response?). I think that 'a file or Unix socket' is incorrect, because we have only one file path here and it is a Unix socket -- one entity, not two different ones.

So, I propose to adjust the existing description with a goal of the admin socket. For example,

> Configure the administration console.
>
> `tt connect` is a client to the console.
>
> <..details, if needed..>

[^1]: The code is more or less straightforward, you can glance on it [here](https://github.com/tarantool/tarantool/blob/01915e6d34bbe6c6bdd5042f8a3d1f4333c08975/src/box/lua/config/applier/console.lua).
[^2]: `console.listen()` supports it, but we decided to forbid it due to security concerns. The admin console doesn't support authorization and always execute commands as the `admin` user (with all the permissions). Since there is no authorization, it shouldn't be accessible remotely. So, listening on a TCP port is forbidden.
[^3]: It is not very consistent with all the other listening socket options, but Mons asked for a simple interface here and I agree that if we support only a Unix socket here, there is no reason to obligate a user to write `unix:` or `unix:/`.